client/argocdServer/application/Application.go

```diff
@@ -419,15 +419,19 @@ func (c ServiceClientImpl) buildPodMetadata(resp *v1alpha1.ApplicationTree, resp
 	var parentWorkflow []string
 	for _, response := range responses {
 		if response != nil && response.Response != nil && response.Request.Kind == "Rollout" {
-			err := json.Unmarshal([]byte(response.Response.Manifest), &rolloutManifest)
+			manifest := make(map[string]interface{})
+			err := json.Unmarshal([]byte(response.Response.Manifest), &manifest)
 			if err != nil {
 				c.logger.Error(err)
 			}
+			rolloutManifests = append(rolloutManifests, manifest)
 		} else if response != nil && response.Response != nil && response.Request.Kind == "Deployment" {
-			err := json.Unmarshal([]byte(response.Response.Manifest), &deploymentManifest)
+			manifest := make(map[string]interface{})
+			err := json.Unmarshal([]byte(response.Response.Manifest), &manifest)
 			if err != nil {
 				c.logger.Error(err)
```

Comment:
continue statement after error log ? or we want empty manifest in array.

Reply:
This is existing logic which I took from replicaSetManifests code Block in this method. 
This can be changed, but impact is unknown.

Reply:
done